build.sbt

```diff
@@ -25,7 +25,7 @@ lazy val library = (project in file("."))
   .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning)
```

Comment:
(Optional) SbtAutoBuildPlugin no longer has to be explicitly enabled (since 3.0.0). See https://github.com/hmrc/sbt-auto-build#version-300

Reply:
Nice spot.